app/assets/stylesheets/thredded/base/_typography.scss

```diff
@@ -39,3 +39,12 @@
   fill: currentColor;
 }
 
+%thredded--quote {
```

Comment:
`thredded--blockquote`, because "quote" usually refers to the inline <q> element.